l2geth/scripts/start.sh

```diff
@@ -6,16 +6,18 @@ REPO=$DIR/..
 IS_VERIFIER=
 ROLLUP_SYNC_SERVICE_ENABLE=true
 DATADIR=$HOME/.ethereum
-TARGET_GAS_LIMIT=11000000
+TARGET_GAS_LIMIT=15000000
 ETH1_CTC_DEPLOYMENT_HEIGHT=12686738
 ROLLUP_CLIENT_HTTP=http://localhost:7878
 ROLLUP_POLL_INTERVAL=15s
-ROLLUP_TIMESTAMP_REFRESH=3m
+ROLLUP_TIMESTAMP_REFRESH=15s
 CACHE=1024
 RPC_PORT=8545
 WS_PORT=8546
 VERBOSITY=3
 ROLLUP_BACKEND=l2
+CHAIN_ID=69
```

Comment:
should we change this for consistency?

Reply:
ROLLUP_TIMESTAMP_REFRESH is 3 minutes in production.